test/thermo/PengRobinson_Test.cpp

```diff
@@ -0,0 +1,312 @@
+    }
+}
+
+TEST_F(PengRobinson_Test, setTP)
+{
+    // Check to make sure that the phase diagram is accurately reproduced for a few select isobars
+
+    // All sub-cooled liquid:
+    const double p1[6] = {
+        1.7084253549322079e+002,
+        1.6543121742659784e+002,
+        1.6066148681014121e+002,
+        1.5639259178086871e+002,
+        1.525268502259365e+002,
+        1.4899341020317422e+002
+    };
+    // Phase change between temperatures 4 & 5:
+    const double p2[6] = {
+        7.3025772038910179e+002,
+        7.0307777665949902e+002,
+        6.7179381832541878e+002,
+        6.3389192023192868e+002,
+        5.8250166044528487e+002,
+        3.8226318921022073e+002
+    };
+    // Supercritical; no discontinuity in rho values:
+    const double p3[6] = {
+        7.8626284773748239e+002,
+        7.6743023689871097e+002,
+        7.4747463917603955e+002,
+        7.2620055080831412e+002,
+        7.0335270498118734e+002,
+        6.7859003092723128e+002
+    };
+
+    for(int i=0; i<6; ++i)
+    {
+        const double temp = 294 + i*2;
+        set_r(0.999);
+        test_phase->setState_TP(temp, 5542027.5);
+        EXPECT_NEAR(test_phase->density(),p1[i],1.e-8);
+
+        test_phase->setState_TP(temp, 7388370.);
+        EXPECT_NEAR(test_phase->density(),p2[i],1.e-8);
+
+        test_phase->setState_TP(temp, 9236712.5);
+        EXPECT_NEAR(test_phase->density(),p3[i],1.e-8);
+    }
+}
+
+TEST_F(PengRobinson_Test, getPressure)
+{
+    // Check to make sure that the P-R equation is accurately reproduced for a few selected values
+
+    /* This test uses CO2 as the only species (mole fraction 99.9%, balance H2).
+    *  Values of a_coeff, b_coeff are calculated based on the the critical temperature and pressure values of CO2 as follows:
+    *       a_coeff = 0.457235(RT_crit)^2/p_crit
+    *       b_coeff = 0.077796(RT_crit)/p_crit
+    *  The temperature dependent parameter in P-R EoS is calculated as
+    *       \alpha = [1 + \kappa(1 - sqrt{T/T_crit}]^2
+    *  kappa is a function calulated based on the accentric factor.
+    */
+
+    double a_coeff = 3.958095109E+5;
+    double b_coeff = 26.62616317/1000;
+    double acc_factor = 0.228;
+    double pres_theoretical, kappa, alpha, mv;
+    const double rho = 1.0737; 
+
+    //Calculate kappa value 
+    kappa = 0.37464 + 1.54226*acc_factor - 0.26992*acc_factor*acc_factor;
+
+    for (int i = 0; i<10; i++)
+    {
+        const double temp = 296 + i * 2;
+        set_r(1.0);        
+        test_phase->setState_TR(temp, rho);
+        const double Tcrit = test_phase->critTemperature();
+        mv = 1 / rho * test_phase->meanMolecularWeight();
+        //Calculate pressure using Peng-Robinson EoS
+        alpha = pow(1 + kappa*(1 - sqrt(temp / Tcrit)), 2.0);
+        pres_theoretical = GasConstant*temp / (mv - b_coeff) - a_coeff*alpha / (mv*mv + 2*b_coeff*mv - b_coeff*b_coeff);

```

Comment:
Can you limit lines to ~80 characters? This happens a few times in the C++ test files.

Reply:
Corrected.